ajax/libs/gitment/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "gitment",
+  "filename": "gitment.browser.js",
+  "version": "0.0.3",
+  "description": "A comment system based on GitHub Issues",
+  "author": {
+    "name": "Shiquan Sun",
+    "url": "https://github.com/imsun"
+  },
+  "keywords": [
+    "comment system",
+    "GitHub Issues"
```

Comment:
Let's add more keywords. This would help people find this lib easier.
![2018-02-03 12 36 23](https://user-images.githubusercontent.com/8349587/35763080-0bfc9bee-08df-11e8-9cb3-13004a98296b.png)
Thank you.